src/main/java/com/minecolonies/coremod/colony/buildings/modules/AbstractCraftingBuildingModule.java

```diff
@@ -67,8 +220,481 @@ public void serializeToView(PacketBuffer buf)
         buf.writeBoolean(this.canLearnCraftingRecipes());
         buf.writeBoolean(this.canLearnFurnaceRecipes());
         buf.writeBoolean(this.canLearnLargeRecipes());
+
+        final List<IRecipeStorage> storages = new ArrayList<>();
+        final Map<ResourceLocation, CustomRecipe> crafterRecipes = CustomRecipeManager.getInstance().getAllRecipes().getOrDefault(building.getJobName() + "_" + getId(), Collections.emptyMap());

```

Comment:
++Miralderps I think, but this should be using `getCustomRecipeKey()` rather than manually concatenating the job and id.